add SoFi case study (#366)
sourcegraph/about: blogposts/sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph.md

---
title: SoFi moves fast on hundreds of microservices with Sourcegraph
author: Vanesa Ortiz
publishDate: 2019-11-19T10:00-07:00
tags: [
  blog
]
slug: sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph

> heroImage: https://about.sourcegraph.com/case-studies/sofi-sourcegraph-case-study.png

Reviewer: I would consider using one of the iphone with financial info images from the home page, as it's not clear what the woman on the swing symbolizes to me.

Author: OK, I'll ask Victoria to update it.

published: true
description: "With Sourcegraph, SoFi can innovate and move quickly while keeping up with hundreds of microservices."
---

<p style="text-align: center">

>   <img src="https://localhost:9000/case-studies/sofi-sourcegraph-case-study-og-embed.png" />

Reviewer: ```suggestion
  <img src="/case-studies/sofi-sourcegraph-case-study-og-embed.png" />
```

</p>

> SoFi helps its over 900,000 members achieve financial independence to realize their ambitions with products for borrowing, saving, spending, investing, and protecting their money. 

Reviewer: ```suggestion
SoFi empowers its 900,000+ members to achieve their financial goals and independence through borrowing, saving, spending, and investing.
```

I suggest removing the "protecting their money" part as it felt redundant.

Author: This line was specifically added by the the director of communications at SoFi, so I'm hesitant to change it.

> With Sourcegraph, SoFi can innovate and move quickly while keeping up with [hundreds of microservices](/case-studies/sofi-moves-fast-on-hundreds-of-microservices).

Reviewer: ```suggestion
In this case study, learn how Sourcegraph enables SoFi to innovate and move quickly while ensuring production stability for their [hundreds of microservices](/case-studies/sofi-moves-fast-on-hundreds-of-microservices).
```
